From aff367791fdd6535021fd0c95ecf2067363073c2 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Tue, 31 Aug 2010 15:49:00 +0000
Subject: Fix command-line help glitch

nautilus-sendo --help says

Usage:
nautilus-sendto [OPTION...] [FILES...] nautilus-sendto

The second 'nautilus-sendto' here is redundant and confusing. With the
patch below, it says just

Usage:
nautilus-sendto [OPTION...] [FILES...]
which is the intended output

https://bugzilla.gnome.org/show_bug.cgi?id=627849
---
diff --git a/src/nautilus-sendto-command.c b/src/nautilus-sendto-command.c
index 4dcbb1a..722386c 100644
--- a/src/nautilus-sendto-command.c
+++ b/src/nautilus-sendto-command.c
@@ -804,7 +804,7 @@ int main (int argc, char **argv)
 	textdomain (GETTEXT_PACKAGE);
 
 	g_thread_init (NULL);
-	context = g_option_context_new ("nautilus-sendto");
+	context = g_option_context_new ("");
 	g_option_context_add_main_entries (context, entries, GETTEXT_PACKAGE);
 	g_option_context_add_group (context, gtk_get_option_group (TRUE));
 	if (g_option_context_parse (context, &argc, &argv, &error) == FALSE) {
--
cgit v0.8.3.1
